.github/workflows/lint.yml

```diff
@@ -26,6 +26,27 @@ jobs:
           sudo rm -rf /opt/ghc
           sudo rm -rf "/usr/local/share/boost"
           sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+
+      - name: Install LLVM 18
+        run: |
+          echo "Installing LLVM 18 on Ubuntu"
+          sudo apt-get update
+          sudo apt-get install -y lsb-release wget software-properties-common
+          wget https://apt.llvm.org/llvm.sh
+          chmod +x llvm.sh
+          sudo ./llvm.sh 18
+
+      - name: Set LLVM_SYS_180_PREFIX
+        run: |
+          prefix=$(llvm-config-18 --prefix)
+          echo "LLVM_SYS_180_PREFIX=$prefix" >> $GITHUB_ENV
+
+      - name: Verify LLVM Installation
+        run: |
+          echo "Using LLVM version:"
+          llvm-config-18 --version
+          echo "LLVM_SYS_180_PREFIX=$LLVM_SYS_180_PREFIX"
+
```

Comment:
move to seperate github actions

Reply:
Is either build or release a better place for llvm configuration?

Reply:
or make the new seperate github actions such as install.yml or llvm.yml